### ISSUE TITLE
MIMIC-CXR JPG dataset resize

### ISSUE THREAD

**Issue by njan-creative** [MIT-LCP/mimic-code#1436]
99% of papers use 512x512 or maximum 1024x1024 size images in chest x-ray dataset.
512x512 after resize would hardly be 6-7 GB after zipping.

it is criminal waste of resources to have people download data of 600GB and preprocess it to 6 GB.
Can you please resize and upload..

**Comment by alistairewj** [comment]
I agree it would be reasonable to have a minified version of the dataset available but it's not top priority for us right now. Maybe in the new year!

**Comment by njan-creative** [comment]
Thanks. Waiting for the same..

**Comment by njan-creative** [comment]
Happy new year.. Alistair Johnson..

@alistairewj
